docs/release-notes/v11.8.0.md

```diff
@@ -0,0 +1,44 @@
+---
+id: v11.8.0
+title: Version 11.8.0 Release Notes
+sidebar_label: v11.8.0
+image: "img/release-note-link-preview.png"
+---
+
+_Oct 26th, 2023_
+
+## Console
+
+### New Features
```

Comment:
Didn't we say to try and use these titles to describe features in order to ahve them nicely indexed in the page TOC on page side?